### PR TITLE
feat:Added deleting an edit functionality

### LINKED ISSUE #167: Cancel button when editing project details from Dashboard
When the user is on the Dashboard tab for a project and clicks the "Edit Details" button, StatWrap opens up a way to modify the project details as well as the project tags.  The only available option is to "Save Details".  However, users should have a way to cancel any edits, and revert them.

1. Add a "Cancel" button to the right of the "Save Details" button
2. If the "Cancel" button is clicked, any edits should be discarded
3. After clicking "Cancel", the user should exit the editing interface and return to the view of the Dashboard

Ideally if the user had made changes, when they click "Cancel", step 2 should prompt the user "You have made changes.  Do you wish to discard those changes?"  This provides an extra protection if the user clicked Cancel by accident.

image

### PR DESCRIPTION
**Pull request by Gmin2**
image
fixes #167 

Added functionality to delete edited changes

### CODE REVIEW

File: app/components/Project/About/About.js [hunk 1]
```diff
@@ -106,7 +108,37 @@ const about = props => {
...
     }
   };
+  const handleCancel = () => {
+    if (unsavedChanges) {
+      const confirmCancel = dialog.showMessageBoxSync({
+        type: 'question',
+        buttons: ['Yes', 'No'],
+        title: 'Confirm Cancel',
+        message: 'You have unsaved changes. Do you wish to discard those changes?'
+      });
```
Comment (Abhijay007) [inline app/components/Project/About/About.js:118]: Hey @utnim2, your work looks good overall, but it seems like the dialog box isn't functioning properly. Could you please look into this this? I suspect you might have missed importing the dialog box somewhere.

Comment (utnim2) [reply app/components/Project/About/About.js:118]: Thanks for your review @Abhijay007 ,looking into it

Comment (lrasmus) [reply app/components/Project/About/About.js:118]: Yes thanks @Abhijay007 for the review!  @utnim2, I think also we're missing the call to ` setUnsavedChanges(true);`, so `unsavedChanges` is always `false`.

Comment (utnim2) [reply app/components/Project/About/About.js:118]: @lrasmus was working on this only. Pushing some changes in some moments

**Comment by Abhijay007** [comment]
Hi @utnim2 just checking in if you are still working on this PR, If yes, that's fine. Please let me know if you need any assistance. If not, please let me know if I can proceed further with this issue.

**Comment by utnim2** [comment]
> Hi @utnim2 just checking in if you are still working on this PR, If yes, that's fine. Please let me know if you need any assistance. If not, please let me know if I can proceed further with this issue.

working on it

**Comment by lrasmus** [comment]
@Gmin2 - were you still interested in working on this?  If not, no worries!  I think we can accept the work that you have done and can have someone else finish up the remaining parts.  Thanks!

**Comment by Abhijay007** [comment]
> @Gmin2 - were you still interested in working on this? If not, no worries! I think we can accept the work that you have done and can have someone else finish up the remaining parts. Thanks!
 
 
>> @Gmin2 - were you still interested in working on this? If not, no worries! I think we can accept the work that you have done and can have someone else finish up the remaining parts. Tha


Hi @lrasmus as @Gmin2 mentioned that he is no longer working on this issue, can I take this forward?

**Comment by lrasmus** [comment]
Not a problem @Gmin2!  I appreciate everything you've done on this.  Because this is in progress, @Gmin2 could you [update the PR to use the develop branch](https://github.blog/2016-08-15-change-the-base-branch-of-a-pull-request/)?  

@Abhijay007 really appreciate you stepping in - I'll assign #167 to you, and if you want to finish this up that'd be amazing.

**Comment by lrasmus** [comment]
Superseded by @Abhijay007's PR #207

**Pull request closed without merging**